content/src/content/jcr_root/apps/core/wcm/components/teaser/v1/teaser/clientlibs/editor/js/teaser.js

```diff
@@ -47,6 +47,16 @@
                 retrievePageInfo($dialogContent);
             });
 
+            var $titleTextfield = $dialogContent.find(titleTextfieldSelector);
+            $titleTextfield.on("change", function() {
```

Comment:
@bpauli - shouldn't we add this change handling internally to the `checkboxTextfieldTuple`? rather than exposing `CheckboxTextfieldTuple.prototype.updatePreviousTextValue`?

Would be good to also make sure we've covered this case with a UI test and check that the other component dialogs that use the `CheckboxTextfieldTuple` are still working correctly (download and image).